native/src/routes/__tests__/Landing.spec.tsx

```diff
@@ -12,10 +11,12 @@ import wrapWithTheme from '../../testing/wrapWithTheme'
 import { checkLocationPermission, requestLocationPermission } from '../../utils/LocationPermissionManager'
 import Landing from '../Landing'
 
+const mockTheme = buildConfig().lightTheme
+
 jest.mock('react-i18next')
 jest.mock('styled-components', () => ({
   ...jest.requireActual('styled-components'),
-  useTheme: () => buildConfig().lightTheme
+  useTheme: () => mockTheme
 }))
```

Comment:
Perhaps we could create a static mock for `styled-components`, lots of duplicated code. Oc no need to do this here.

Reply:
https://issues.tuerantuer.org/browse/IGAPP-896